### PR TITLE
New card form

### PR DESCRIPTION
**Pull request by yuda1124**
edit eslint test only js file from all file.
add event images from server and edit date form.
fix error and warning!

### CODE REVIEW

File: shared/components/headerbox.js [hunk 1]
```diff
@@ -126,7 +128,9 @@ export default class Headerbox extends Component {
               index: this.props.tabview_index,
               routes: this.props.tabview_routes
             }}
-            renderScene={()=>{}}
+            renderScene={(props) => {
+              return <TabViewPage {...props} renderScene={()=>{}}/>;
```
Comment (ochanje210) [inline shared/components/headerbox.js]: TabViewPage is deprecated and plz remove it.

File: shared/components/headerbox.js [hunk 2]
```diff
@@ -15,10 +15,10 @@ const styles = StyleSheet.create({
     marginLeft: 0,
     marginTop: 24,
     marginBottom: 10,
-    color: "#2b2b2b",
+    color: '#2b2b2b',
     ...Platform.select({
-      android: {
-        fontFamily: "Roboto-Medium"
+      andorid: {
```
Comment (ochanje210) [inline shared/components/headerbox.js]: u have brought the typo back

**Comment by mun625** [comment]
LGTM

**Comment by elegant651** [comment]
LGTM

**Comment by ochanje210** [comment]
never mind, I forgot to update the tabview module version on package.json. I will apply it on the next PR

**Pull request merged**